strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/cron/jobs/RebuildMavenMetadataCronJobTestIT.java

```diff
@@ -47,15 +45,14 @@
 public class RebuildMavenMetadataCronJobTestIT
         extends BaseCronJobWithMavenIndexingTestCase
 {
-
-    private static final String[] CLASSIFIERS = { "javadoc",
-                                                  "sources",
-                                                  "source-release" };
-
+	
+	private static final String RMMCJTIT_SNAPSHOTS = "rmmcjtit-snapshots";
+	
+	private static final String TRMIR_SNAPSHOTS = "trmir-snapshots";
```

Comment:
Cosmetic: Please, follow our coding convention. You should use four spaces here.